packages/components/src/tooltip/index.js

```diff
@@ -119,7 +119,9 @@ class Tooltip extends Component {
 	createToggleIsOver( eventName, isDelayed ) {
 		return ( event ) => {
 			// Preserve original child callback behavior
-			this.emitToChild( eventName, event );
+			if ( event.type !== 'mousedown' ) {
```

Comment:
What about `click` ? Noting since it's mentioned in the text of the original comment.